app/medusa/src/main.rs

```diff
@@ -21,9 +21,6 @@ fn main() -> ! {
 
     const CYCLES_PER_MS: u32 = 400_000;
 
-    #[cfg(feature = "traptrace")]
-    kern::profiling::configure_events_table(tracing::table());
```

Comment:
I don't think we wanted to drop this?

Reply:
I suspect we aren't likely to use it, maybe replace it with a comment mentioning we could add it here